integration-tests/docker/docker-compose.test-env.yml

```diff
@@ -0,0 +1,173 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Do we need both integration-tests/docker/docker-compose.test-env.yml and integration-tests/docker/docker-compose.yml?

Reply:
The issue is that we have 2 different configuration to run integration tests.
security tests required: "DRUID_INTEGRATION_TEST_GROUP" env variables
s3, azure, etc required: "OVERRIDE_ENV"

I can merge both this ways but then I will need to create file with empty configuration in (environment-configs/override-examples/)

Reply:
I'm a little confused. When is integration-tests/docker/docker-compose.yml use? 
I see that s3, azure, gcs uses docker-compose.override-env.yml and everything else uses docker-compose.test-env.yml.
I think it will be very helpful if you can list out the purpose of each yml

Reply:
Also I think every tests needs DRUID_INTEGRATION_TEST_GROUP

Reply:
1. I expect to use "docker-compose.yml" for all tests that do not need custom env configs
But unfortunately this is not works, because druid.sh prepare data for "query" and "security" tests (this is wrong because this must be done by tests)

I think that we can remove "docker-compose.test-env.yml"
I move DRUID_INTEGRATION_TEST_GROUP to docker-compose.yml

2. docker-compose - it's files for containers, not for tests. If your tests need "DRUID_INTEGRATION_TEST_GROUP" variable you pass this variable my maven run or by some config file.

Reply:
removed docker-compose.test-env.yml
Added DRUID_INTEGRATION_TEST_GROUP env to docker-compose.yml